### PR TITLE
Fix some wording and spelling

### PR DESCRIPTION
**Pull request by DelazJ**
also refs https://github.com/qgis/qgis3_UIX_discussion/issues/19#issuecomment-266994315

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: src/ui/qgspgnewconnectionbase.ui [hunk 1]
```diff
@@ -265,7 +265,7 @@
 &lt;body&gt;
 &lt;p&gt;When the layer is setup various metadata is required for the PostGIS table. This includes information such as the table row count, geometry type and spatial extents of the data in the geometry column. If the table contains a large number of rows determining this metadata is time consuming.&lt;/p&gt;
 &lt;p&gt;By activating this option the following fast table metadata operations are done:&lt;/p&gt;
-&lt;p&gt;1) Row count is determined from table statistics obtained from running the PostgreSQL table analyze function.&lt;/p&gt;
+&lt;p&gt;1) Row count is determined from results of running the PostgreSQL Analyze function on the table.&lt;/p&gt;
```
Comment (nyalldawson) [inline src/ui/qgspgnewconnectionbase.ui]: This is not quite the same meaning - I believe the important point the original string is trying to convey that the row count is taken from the statistics table, not calculated exactly.

Comment (DelazJ) [reply src/ui/qgspgnewconnectionbase.ui]: Ok i think i'll revert the changes on this though i might confess i'm not comfortable with the end of the sentence. But i'm not a native speaker and might be translating. 
@sveinki can you please elaborate on what made you report an issue on this sentence in https://github.com/qgis/qgis3_UIX_discussion/issues/19#issuecomment-266994315? Thanks

Comment (sveinki) [reply src/ui/qgspgnewconnectionbase.ui]: I guess it was the "analyse" vs "analyze" inconsistncy ? Seems right to me now.

**Comment by DelazJ** [comment]
May I?

**Pull request merged**